test/boost/sstable_datafile_test.cc

```diff
@@ -3291,3 +3291,56 @@ SEASTAR_TEST_CASE(test_reclaimed_bloom_filter_deletion_from_disk) {
         BOOST_REQUIRE(!file_exists(filter_path).get());
     });
 }
+
+SEASTAR_TEST_CASE(test_bloom_filter_reclaim_during_reload) {
+    return test_env::do_with_async([] (test_env& env) {
+#ifndef SCYLLA_ENABLE_ERROR_INJECTION
+        fmt::print("Skipping test as it depends on error injection. Please run in mode where it's enabled (debug,dev).\n");
+#else
```

Comment:
Instead of `#else`, just add a `return` above to short-circuit the test when error injection is not available.

Reply:
Done.